src/invidious/videos/caption.cr

```diff
@@ -31,6 +31,61 @@ module Invidious::Videos
       return captions_list
     end
 
+    def timedtext_to_vtt(timedtext : String, tlang = nil) : String
+      # In the future, we could just directly work with the url. This is more of a POC
+      cues = [] of XML::Node
+      tree = XML.parse(timedtext)
+      tree = tree.children.first
+
+      tree.children.each do |item|
+        if item.name == "body"
+          item.children.each do |cue|
+            if cue.name == "p"
+              if !(cue.children.size == 1 && cue.children[0].content == "\n")
```

Comment:
those two if statements can be combined with `&&`

Reply:
Whoops! Didn't even notice that.